src/AspNet.Security.OAuth.GitHub/GitHubAuthenticationHandler.cs

```diff
@@ -63,7 +63,7 @@ public class GitHubAuthenticationHandler : OAuthHandler<GitHubAuthenticationOpti
                 var address = await GetEmailAsync(tokens);
                 if (!string.IsNullOrEmpty(address))
                 {
-                    identity.AddClaim(new Claim(ClaimTypes.Email, address, Options.ClaimsIssuer));
+                    identity.AddClaim(new Claim(ClaimTypes.Email, address, ClaimValueTypes.String, Options.ClaimsIssuer));
```

Comment:
I don't think it's necessary.

Reply:
In the similar code for the Vkontakte provider I'm going to use `ClaimValueTypes.Email` constant. Am I also wrong?

Reply:
@Vertigo093i I use `ClaimTypes.Email`, it's claims.

Reply:
I'm talking about the third parameter, not the first one. Value type also has a 'e-mail' type.

Reply:
It's an oopsie on my part (the third parameter is the claim value type, not the issuer).
I fixed it directly in the experimental branch.

Reply:
Note: I used `ClaimValueTypes.String` instead of `ClaimValueTypes.Email` for consistency with the ASP.NET Core 2.0 providers developed by MSFT, that all use `String` for the email claim.

Reply:
Oops, didn't notice this by myself when looking for overloads.

Reply:
OK, to be consistent with others I will also use `String`.